src/rgw/rgw_http_errors.h

```diff
@@ -10,69 +10,69 @@ struct rgw_http_errors {
   int err_no;
   int http_ret;
   const char *s3_code;
+  const char *s3_message; // error message in DeleteMultiObjects.
```

Comment:
We're now [using s->err.message](https://github.com/ceph/ceph/blob/master/src/rgw/rgw_common.cc#L356) to record error message.